tree/30_generic_methods/log_rudder.cf

```diff
@@ -203,7 +203,7 @@ bundle agent _log_rudder_v2(expected_reports_source, message, class_prefix, args
 
   methods:
     report_data_found::
-      "any" usebundle => _rudder_common_reports_generic("${report_data[1]}", "${class_prefix}", "${report_data[3]}", "${promisers}", "${args_line1}", "${message_line1}"),
+      "any" usebundle => _rudder_common_reports_generic("${report_data[1]}", "${class_prefix}", "${report_data[3]}", "${promisers}", "${args_line1}", "${message_line1}",""),
```

Comment:
Maybe there should be a value here, i dunno ... but since logger_v2 is almost never called ...